bot/cogs/utils.py

```diff
@@ -257,6 +257,24 @@ def check(m: Message) -> bool:
         embed.description = best_match
         await ctx.send(embed=embed)
 
+    @command(aliases=("poll",))
+    @with_role(*MODERATION_ROLES)
+    async def vote(self, ctx: Context, title: str, *options: str) -> None:
+        """
+        Build a quick voting poll with matching reactions with the provided options.
+
+        A maximum of 20 options can be provided, as Discord supports a max of 20
+        reactions on a single message.
+        """
+        if len(options) > 20:
+            raise BadArgument("I can only handle 20 options!")
+
+        options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=127462)}
+        embed = Embed(title=title, description="\n".join(options.values()))
+        message = await ctx.send(embed=embed)
+        for reaction in options:
+            await message.add_reaction(reaction)
```

Comment:
Is it worth using `asyncio.gather()`? Maybe not given the rate limits.

Reply:
Not at all because in the end it's still limited by the lib and we're not doing anything afterwards such as waiting for reaction adds from users.